examples/restaurant_demo/restaurant_demo.ipynb

```diff
@@ -0,0 +1,3257 @@
+{
```

Comment:
<p>No need to print this, or keep it as a separate cell </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p> It is easier to understand for a complete newbie if they are looking into the code. Might not be need for the experienced users. But as we are making the demo for just anyone interested it would be easier if they have  idea of both the private and non-private count values.Yes graph does that part too but this should help too. I think Jupyter Notebook serves that part really well to make it easy to inspect or check the results. </p><p> </p><p> </p><p> </p><p> </p>